optuna/study.py

```diff
@@ -175,6 +175,31 @@ def __setstate__(self, state: Dict[Any, Any]) -> None:
     def user_attrs(self) -> Dict[str, Any]:
         """Return user attributes.
 
+        Please see the example code of the :func:`~optuna.study.Study.set_user_attr` method.
```

Comment:
I commented in https://github.com/optuna/optuna/pull/1744#discussion_r482058080, how about using `.. seealso::`?
```suggestion
        .. seealso
        
            See :func:`~optuna.study.Study.set_user_attr` for related method.
```

Reply:
@HideakiImamura  Thanks for nice idea!